packages/d3fc-series/src/canvas/multi.js

```diff
@@ -18,15 +18,19 @@ export default () => {
                 .xScale(xScale)
                 .yScale(yScale);
 
-            const adaptedDecorate = dataSeries.decorate();
-            dataSeries.decorate((c, d, i) => {
-                base.decorate()(c, data, index);
-                adaptedDecorate(c, d, i);
-            });
-
-            dataSeries(seriesData);
-
-            dataSeries.decorate(adaptedDecorate);
+            if (dataSeries.decorate) {
+                const adaptedDecorate = dataSeries.decorate();
+                dataSeries.decorate((c, d, i) => {
+                    base.decorate()(c, data, index);
```

Comment:
I'm thinking we should still call `base.decorate` in the case that `dataSeries.decorate` is not defined. What do you think?

Reply:
I don't see any issues with this